.changeset/young-spoons-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/vercel': minor
```

Comment:
```suggestion
'@astrojs/vercel': patch
```
A minor release usually means there is something new for users to read up on.

Reply:
Well this does allow for you to return a Promise from the middleware and you previously couldn't do that. So, as long the version increases somewhere, I think it's fine.

Reply:
What Arsh means is that the version will be increased automatically by a bot in an future PR, so you only have to specify the change type now

Reply:
I'm just not sure what or if the version will change when the changeset is set to a patch. Will it theoretically go from 6.0.0 => 6.0.1?

Reply:
I think another patch has been released so it's already 6.0.1 but yeah, in a release PR, it will be bumped to 6.0.2 as well as other packages depending on it